### DISCUSSION TITLE
Error 3 files

### DISCUSSION THREAD

**Discussion by Haki-Skykid** [thatskymod/Sky-CotL-Scripts#83] [General]
Currently 3 files in libs do not work with the new version of canvas I hope you will fix them soon
![Screenshot_20240511-190650_Canvas](https://github.com/thatskymod/Sky-CotL-Scripts/assets/168264875/d60c07ba-aa64-48a6-99e0-412af98bd7b5)
![Screenshot_20240511-190650_Canvas](https://github.com/thatskymod/Sky-CotL-Scripts/assets/168264875/5e634e72-9fe4-4829-bafc-6aeba3c34043)

**Comment by thatskymod** [comment]
Okay, but first, let me clarify that as a collective repository, I won't be fixing the bugs personally. I will provide fixes as soon as they become available through messenger channels or the authors' official resources, which I regularly check. But I see your point. I have already scrapped the problematic files from the Libs folder and replaced them with new, tested files. This is not the final outcome, so enjoy the current update and stay tuned for more!

**Reply by Haki-Skykid** [reply]
Thank You❤

**Comment by thatskymod** [comment]
### Update finished
All files from the [Libs/](https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas/Libs) folder have now been replaced by their newer counterparts.

Thank you for standing by while the issues were being addressed.
